conductr_cli/main_handler.py

```diff
@@ -19,7 +19,7 @@ def run(callback):
 
         # Ensure log dir is present before errors are being logged
         log_dir = os.path.abspath(os.path.join(DEFAULT_ERROR_LOG_FILE, '..'))
-        os.makedirs(log_dir)
+        os.makedirs(log_dir, exist_ok=True)
```

Comment:
This is now done as separate commit: https://github.com/typesafehub/conductr-cli/pull/182/commits/a1ebd41e3f7864536c20074c8ad660e3e395cab3